### DISCUSSION TITLE
Help find OSM edit now causing OsmAnd "flooding"

### DISCUSSION THREAD

**Discussion by thompehw** [osmandapp/OsmAnd#12345] [Maps Related]
I noticed a case of "flooding" at some zoom levels in OsmAnd around geo:51.07450,1.16528

![Screenshot_20210802-041535_OsmAnd+](https://user-images.githubusercontent.com/39751382/127800665-27ac17db-17d0-4f60-bf99-09c23f17f043.jpg)

It doesn't appear with the England South East map of 1st July, only when I have live updates. So I assume that something was edited in OSM since 1st July that is causing a problem. I had a look at the OSM changes using achavi but I haven't found the cause yet. Can anyone help? Thanks!

**Comment by thompehw** [comment]
There was an edit to this section of coastline during that timeframe in [Changeset 107363707](https://www.openstreetmap.org/changeset/107363707) but I can't see a problem. 

Osm Inspector (Coastline mode) gives no errors.

Should I raise this as an Issue (potential OsmAnd bug)? If no-one says otherwise then in a few days I will.

Thanks!

**Comment by tommycrock** [comment]
I haven't looked into this but was just looking at the north end of the Forth bridge and spotted the same. No idea if it's related but thought I'd mention it in case it helps in bug finding.
![Screenshot_20210806-064954_OsmAnd+](https://user-images.githubusercontent.com/5749884/128470218-b2bd4985-cec2-47fa-bfc8-57b2cd3d71da.jpg)

lat=56.0081660675601&lon=-3.390493029728532&z=16

**Reply by thompehw** [reply]
Hhhhm! I tried to replicate your North Queensferry flooding. I downloaded the Scotland map (currently dated 1st July) and couldn't see the flooding at any zoom level. Then I did a live update and still don't see it.

Weird! What date is your Scotland map? (Look in Download_Maps>Local" within the app.)  

I'm using Android Osmand+ 3.9.10, but I'm guessing this is about the maps not the apps.

**Reply by tommycrock** [reply]
Yeah, sorry, it was the May '21 version of the Scotland map.

**Comment by thompehw** [comment]
With the latest map updates there is no problem at the location in my original post. I notice that the devs are investigating - thanks!
